### PR TITLE
[BAQE-1613] Move jbpm-test-coverage to the db jenkins matrix

### PR DESCRIPTION
**Pull request by afalhambra**
**[BAQE-1613](https://issues.redhat.com/browse/BAQE-1613)**: Move jbpm-test-coverage to the db jenkins matrix

**referenced Pull Requests**: 

* https://github.com/jboss-integration/bxms-qe-tests/pull/4040

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: jbpm-test-coverage/src/test/java/org/jbpm/test/functional/timer/GlobalTimerServiceBaseTest.java [hunk 1]
```diff
@@ -803,21 +808,20 @@ public void afterNodeLeft(ProcessNodeLeftEvent event) {
...
             ut.rollback();
             throw ex;
         }
 
-        // now wait for 1 second for first timer to trigger
-        countDownListener.waitTillCompleted();
-        countDownListener.reset(1);
-        
+        // now wait for more than 3 seconds for all timers to be triggered
+        Thread.sleep(5000);
```
Comment (gmunozfe) [inline jbpm-test-coverage/src/test/java/org/jbpm/test/functional/timer/GlobalTimerServiceBaseTest.java:823]: Couldn't it be used the countdown listener instead?

File: jbpm-test-coverage/src/test/java/org/jbpm/test/functional/timer/GlobalTimerServiceBaseTest.java [hunk 2]
```diff
@@ -838,13 +843,12 @@ public void afterNodeLeft(ProcessNodeLeftEvent event) {
         }
 
         // let's wait to ensure no more timers are expired and triggered
-        countDownListener.waitTillCompleted(3000);
-        
+        Thread.sleep(5000);
```
Comment (gmunozfe) [inline jbpm-test-coverage/src/test/java/org/jbpm/test/functional/timer/GlobalTimerServiceBaseTest.java:846]: same

**Review by gmunozfe — APPROVED** [review]
Looks good to me, just a few questions to clarify but overall is awesome work and a great advance for total coverage in db matrix, well done @afalhambra !

**Comment by afalhambra** [comment]
Jenkins retest this

**Comment by afalhambra** [comment]
@mareknovotny this is ready to merge

**Comment by afalhambra** [comment]
Jenkins retest this

**Pull request closed without merging**